docs/installation-laravel.md

```diff
@@ -166,7 +166,13 @@ return [
 ];
 ```
 
-After the config & migration have been published you can create the role- and permission-tables by running the migrations:
+Then, apply the configuration:
+
+```bash
+php artisan config:cache
```

Comment:
Why run config:cache here?

Reply:
without run that command, i got SQL error like i mention before

Reply:
must i change with `config:clear` sir?